bundles/org.openhab.binding.evcc/README.md

```diff
@@ -1,7 +1,7 @@
 # evcc Binding
 
 This binding integrates [evcc - electric vehicle charging control](https://evcc.io), a project that provides a control center for electric vehicle charging.
-The binding requires evcc [version 0.123.1](https://github.com/evcc-io/evcc/releases/tag/0.123.1) or newer and is tested with this version.
+The binding is compatbile to evcc starting with [version 0.123.1](https://github.com/evcc-io/evcc/releases/tag/0.123.1) or newer. Starting with [evcc version 0.125.0](https://github.com/evcc-io/evcc/releases/tag/0.125.0) this version of the binding is mandatory as some of the evcc rest-API parameters got discontinued. This binding was tested with [version 0.125.0](https://github.com/evcc-io/evcc/releases/tag/0.125.0).
```

Comment:
Let’s rephrase it a bit, there is IMO no value in saying that this version of the binding is required, because the user cannot really choose a binding version.

```suggestion
This binding is compatible to evcc [version 0.123.1](https://github.com/evcc-io/evcc/releases/tag/0.123.1) or newer and was tested with [version 0.125.0](https://github.com/evcc-io/evcc/releases/tag/0.125.0).
```

Reply:
Ok, I'll take that over.
A user going from OH4.1.0 to OH4.2.0 stable will anyhow not recognize the details between the different incremental updates.